### ISSUE TITLE
Wrong/useless information in info bubble on sidebar

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#2873]
#### Issue

On sidebar i see shorted name of records ( if are too long)... on hover it shows me, if is record the task, case, opportunity or etc. 
Coud it show me in pop up (info bubble) instead of this name of module the whole name of record. I think it helps better to orientation in records. The name of records you can you see by icon in front of each record name on sidebar.

Videoscreen: http://screencast.com/t/dZ9h3pyBz

#### Expected Behavior

It shows us the full name of record name in pop up (info bubble)

#### Actual Behavior

It shows the name of module to which is record assign.

#### Steps to Reproduce

1. Look on sidebar
2. Go over any of record, which was visited
3. Look on pop up bubble

#### Your Environment

* SuiteCRM Version used: SuiteCRM 7.7.9
* Browser name and version: Chrome 55.0.2883.87 m (64-bit)
* Environment name and version: PHP, MySQL 5.6
* Operating System and version: Win 10 basic

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Wrong/useless information in info bubble on sidebar](https://trello.com/c/wWrrw7N3/42-wrong-useless-information-in-info-bubble-on-sidebar)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
